src/engine/renderer/Material.cpp

```diff
@@ -1031,7 +1031,7 @@ void BindShaderLightMapping( Material* material ) {
 	gl_lightMappingShaderMaterial->SetReliefMapping( material->enableReliefMapping );
 	/* Reflective specular setting is different here than in ProcessMaterialLightMapping(),
 	because we don't have cubemaps built yet at this point, but for the purposes of the material ordering there's no difference */
-	gl_lightMappingShaderMaterial->SetReflectiveSpecular( material->enableSpecularMapping && !( tr.refdef.rdflags & RDF_NOCUBEMAP ) );
+	gl_lightMappingShaderMaterial->SetReflectiveSpecular( glConfig2.reflectionMapping && material->enableSpecularMapping && !( tr.refdef.rdflags & RDF_NOCUBEMAP ) );
```

Comment:
Maybe there is a way to test for `glConfig2.reflectionMapping` when setting `material->enableSpecularMapping` earlier? Anyway, this is not blocking.

Reply:
You can use specular mapping while reflection mapping is disabled. That's the broken case.

Reply:
Ah I see, for some reasons my mind misread `glConfig2.specularMapping` instead. 😅️